myst_nbs/time_series/Air_passengers-Prophet_with_Bayesian_workflow.myst.md

```diff
@@ -6,46 +6,46 @@ jupytext:
     format_version: 0.13
     jupytext_version: 1.13.7
 kernelspec:
-  display_name: pymc3-dev-py38
+  display_name: Python 3
   language: python
-  name: pymc3-dev-py38
+  name: python3
 ---
 
+(Air_passengers-Prophet_with_Bayesian_workflow)=
 # Air passengers - Prophet-like model
 
-We're going to look at the "air passengers" dataset, which tracks the monthly totals of a US airline passengers from 1949 to 1960. We could fit this using the [Prophet](https://facebook.github.io/prophet/) model (indeed, this dataset is one of the examples they provide in their documentation), but instead we'll make our own Prophet-like model in PyMC3. This will make it a lot easier to inspect the model's components and to do prior predictive checks (an integral component of the [Bayesian workflow](https://arxiv.org/abs/2011.01808)).
+:::{post} April, 2022
+:tags: time series, prophet
+:category: intermediate
+:author: Marco Gorelli
```

Comment:
You should add yourself here, we agreed on including everyone who authored, adapted or updated the notebook here to have them appear in the citation advise at the bottom

Reply:
Hi yes, updated. Thanks mate!